tutorials/scripting/signals_concise_example.rst

```diff
@@ -0,0 +1,52 @@
+.. _doc_signals_concise_example:
+
+Signals, concise example
+========================
+
+.. image:: img/signals_concise_example.webp
+
+Goal: Rotate the **yellow gate** by sending a signal from the **red button**.
+
+``red_button.gd``
+
+.. tabs::
+  .. code-tab:: gdscript GDScript
```

Comment:
```suggestion
 .. code-tab:: gdscript GDScript
```
For all, or the format won't work correctly

Reply:
Thank you.
(Still learning the correct procedures)

Initially I used this page to copy the formatting code from, it had the whitespace there - but it seems to work:
https://raw.githubusercontent.com/godotengine/godot-docs/master/tutorials/scripting/resources.rst
https://docs.godotengine.org/en/stable/tutorials/scripting/resources.html

Image snippet of that code from the resources.rst:

<img width="403" alt="Screenshot 2024-06-09 at 18 56 00" src="https://github.com/godotengine/godot-docs/assets/5437697/642f5cfa-c5e6-4d67-8d04-e8bc3440db23">

Reply:
Yes that has one space before `.. code-tab` you have two, you didn't copy it directly

Reply:
Oh my, I tried to be very prudent but it seemed not to be enough.

You have some sharp eyes, nice catch.

I'll fix the errors soon and build it locally to see if it works nicely.